pkg/scheduler/metrics/metrics.go

```diff
@@ -195,11 +195,12 @@ var (
 			StabilityLevel: metrics.ALPHA,
 		},
 	)
-	PreemptionVictims = metrics.NewGauge(
-		&metrics.GaugeOpts{
+	PreemptionVictims = metrics.NewHistogram(
```

Comment:
BTW: do we break the original metrics by changing this to the histogram? This is in the alpha stage, so it implies we don't guarantee the compatibility of the metrics, right? 

But I think we do need a release note for this.

Reply:
@draveness Thanks for the review. I have included a release note and squashed the commits.

Reply:
> BTW: do we break the original metrics by changing this to the histogram? This is in the alpha stage, so it implies we don't guarantee the compatibility of the metrics, right?

That's my understanding as well. @piosz, can you give us some feedback on this?ff

Reply:
According to [stability-classes](https://github.com/kubernetes/enhancements/blob/master/keps/sig-instrumentation/20190404-kubernetes-control-plane-metrics-stability.md#stability-classes),
`Alpha metrics have no stability guarantees; as such they can be modified or deleted at any time. `.

I agree with @draveness release note is necessary and it's enough, even though a little rough.